### ISSUE TITLE
Apple Keyboard Function Keys

### ISSUE THREAD

**Issue by AJ-SA** [hak5darren/USB-Rubber-Ducky#20]
Hi!

I've been trying to find another reliable way to search and open terminal or any other applications in OSX because CMD-SPACE is risky. This is a default shortcut that any use, include me, have already changed it. However, Fn-F4 (Launchpad Key) is always working and easy to search for application from any window or app.

I wonder if there's a way to add Fn-F4 (or what represent the Launchpad). 

Thank you!
AJ

**Comment by tjt263** [comment]
FYI: Mine is just <kbd>F4</kbd> without <kbd>fn</kbd>.

**Comment by AJ-SA** [comment]
@tjt263 Correct, by default, the Mac OS X Keyboard Setting always has <kbd>Fn</kbd> ON by default. Check this:
https://support.apple.com/en-us/HT204436
